engine/apps/zvonok/phone_provider.py

```diff
@@ -111,8 +112,18 @@ def make_verification_call(self, number: str):
         body = None
         speaker = live_settings.ZVONOK_SPEAKER_ID
 
+        if live_settings.ZVONOK_VERIFICATION_TEMPLATE:
```

Comment:
consider adding a set of unit tests for `ZvonokPhoneProvider.make_verification_call`